### DISCUSSION TITLE
build and deploy github pages

### DISCUSSION THREAD

**Discussion by TYB-hue** [community/community#131932] [New to GitHub]
I have a react project and when I want to build and deploy it with GitHub pages it show me this error :- 

ERROR: Type error must be an instance of error.
can someone help me ?
![Screenshot 2024-07-10 013904](https://github.com/user-attachments/assets/b3892cec-f6b4-4dc3-9b94-a67e235f8543)

**Comment by Aditijainnn** [comment]
> I have a react project and when I want to build and deploy it with GitHub pages it show me this error :-
> 
> ERROR: Type error must be an instance of error. can someone help me ?

Try building your react project locally and see if the error occurs. This can help isolate if the issue is specific to the GitHub Pages deployment process.

or
Read this article https://www.freecodecamp.org/news/deploy-a-react-app-to-github-pages/
You can also try a temp workaround:  disable the Github Page and then enable it again in repo setting. The page should be forced to rebuild. 
(don’t try disable the Github Page since the new build could last forever. And your page will report 404 during this time)

**Comment by Sicilianii** [comment]
Had the same problem, here's the link
https://stackoverflow.com/questions/11577147/how-to-fix-http-404-on-github-pages

If you find this as helpful, please mark this comment as an answer :)
